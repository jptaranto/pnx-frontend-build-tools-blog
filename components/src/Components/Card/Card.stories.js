import Component from './card.html.twig'
import { userEvent, within, expect } from '@storybook/test'

const meta = {
  component: Component,
  args: {
    title: `<a href="#">Card title</a>`,
    description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit...',
  },
  play: async ({ canvasElement }) => {
    const canvas = within(canvasElement)
    const link = canvas.getByRole('link')
    await userEvent.click(link)
    await expect(link).toBeInTheDocument()
  },
}
export default meta

export const Card = {}
