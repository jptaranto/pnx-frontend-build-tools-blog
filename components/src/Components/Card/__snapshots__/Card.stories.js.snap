// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Examples/Card Card smoke-test 1`] = `
<div class="card">
  <div class="card__title">
    <a href="#">
      Card title
    </a>
  </div>
  <div class="card__description">
    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam eu turpis molestie, dictum est a, mattis tellus. Sed dignissim, metus nec fringilla accumsan, risus sem sollicitudin lacus.
  </div>
</div>
`;
